From e6644f9e50372e5298c05c2547fba45ee8df83dc Mon Sep 17 00:00:00 2001
From: Xeonacid <h.dwwwwww@gmail.com>
Date: Sun, 10 Sep 2023 18:24:11 +0800
Subject: [PATCH] tests: set slow for
 gobject/{threadtests,dynamictests,performance}

These tests takes the below time on a most powerful RISC-V hardware, which are longer than test_timeout:
351/353 glib:gobject / threadtests                                             OK             61.93s   4 subtests passed
352/353 glib:gobject+performance+no-valgrind / performance                     OK             85.33s
353/353 glib:gobject / dynamictests                                            OK             101.96s   2 subtests passed

Signed-off-by: Xeonacid <h.dwwwwww@gmail.com>
---
 gobject/tests/meson.build             | 4 ++--
 gobject/tests/performance/meson.build | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/gobject/tests/meson.build b/gobject/tests/meson.build
index 23f131b5c8..0333e48ebc 100644
--- a/gobject/tests/meson.build
+++ b/gobject/tests/meson.build
@@ -78,8 +78,8 @@ gobject_tests = {
   'references' : {},
   'basic-signals' : {},
   'singleton' : {},
-  'threadtests' : {},
-  'dynamictests' : {},
+  'threadtests' : {'suite' : ['slow']},
+  'dynamictests' : {'suite' : ['slow']},
   'binding' : {},
   'bindinggroup' : {},
   'properties' : {},
diff --git a/gobject/tests/performance/meson.build b/gobject/tests/performance/meson.build
index 2984a6c284..5aa720950c 100644
--- a/gobject/tests/performance/meson.build
+++ b/gobject/tests/performance/meson.build
@@ -1,5 +1,5 @@
 gobject_tests = {
-  'performance' : { 'args' : [ '--seconds', '0' ] },
+  'performance' : { 'args' : [ '--seconds', '0' ], 'suite' : ['slow'] },
   'performance-threaded' : { 'args' : [ '--seconds', '0' ] },
 }
 
-- 
GitLab

